cprover_bindings/src/goto_program/location.rs

```diff
@@ -58,6 +67,9 @@ impl Location {
                 format!("<{}>", function_name)
             }
             Location::Loc { file, line, .. } => format!("{}:{}", file, line),
+            Location::Property { property_class, comment, file, line, .. } => {
+                format!("{:?}:{:?},{:?}:{:?}", property_class, comment, file, line)
```

Comment:
I'm fairly certain, given this function's comment, this should only be printing the file/line.